src/core/cs2_sdk/schema.h

```diff
@@ -54,6 +56,45 @@ inline uint64_t hash_64_fnv1a_const(const char *str, const uint64_t value = val_
 }
 
 namespace schema {
+static std::unordered_set<std::string> CS2BadList({
```

Comment:
I'm down for a better idea, it was simple like this currently

Reply:
maybe a normal `vector` instead of `unordered_set` could do better as that's faster